dockstore-common/src/main/scala/io/dockstore/common/WdlBridge.scala

```diff
@@ -105,15 +105,19 @@ class WdlBridge {
     bundle.allCallables.foreach(callable => {
       callable._2 match {
         case w: WorkflowDefinition => {
-          val before = callable._2.asInstanceOf[WorkflowDefinition].meta.mapValues(_.asInstanceOf[MetaValueElementString].value)
-          val metadata = JavaConverters.mapAsJavaMap(before)
+          val workflow_instance = callable._2.asInstanceOf[WorkflowDefinition]
```

Comment:
Was this changed for readability? Trying to see what's different.

Reply:
Yes it was changed to appease Codacy readability rules. https://app.codacy.com/gh/dockstore/dockstore/pullRequest?prid=6994541&bid=22353713. I'm still not quite sure what it is asking for.

Reply:
I see. I looked and get their overall point, but yes, they're still complaining and it's not clear to me why.

Just noticed you are using Python/snake case for the the val names; Most Scala/Java use camelCase and I'm surprised Checkstyle isn't complaining about that -- maybe it's only enabled for Java.

workflow_instance => workflowInstance
converted_workflow_map => convertedWorkflowMap

Reply:
Now using camel case